### ISSUE TITLE
StrictCallSniff reports correct code

### ISSUE THREAD

**Issue by radimvaculik** [slevomat/coding-standard#984]
When I use code:
```
array_keys([]);
```
everything is [OK](https://github.com/slevomat/coding-standard/blob/master/tests/Sniffs/Functions/data/strictCallNoErrors.php#L37). 

But when I use code:
```
array_keys(
    [],
);
```

it is reported as `SlevomatCodingStandard.Functions.StrictCall.StrictParameterMissing`.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/d1068138f274a7e14be561a93f4922407cf1ea51
